src/main/java/Bro.java

```diff
@@ -0,0 +1,110 @@
+                boolean isValidated2 = validateTask(tasks.size(), indexOfUnmarkedTask);
+                if (isValidated2) {
+                    reply = new StringBuilder(" Marked " + tasks.get(indexOfUnmarkedTask) + " as not done.");
+                    tasks.get(indexOfUnmarkedTask).setUncompleted();
+                }
+                else {
+                    reply = new StringBuilder(TASK_DOES_NOT_EXIST);
+                }
+                break;
+            case "todo":
+                Task todo = new ToDo(line);
+                tasks.add(todo);
+                reply = new StringBuilder(" added: " + todo);
+                break;
+            case "deadline":
+                StringBuilder deadlineName = new StringBuilder();
+                StringBuilder by = new StringBuilder();
+                int indexOfDeadline = Arrays.asList(words).indexOf("/by");
+                for (int i = 1; i < indexOfDeadline; ++i) {
+                    deadlineName.append(" ").append(words[i]);
+                }
+                for (int i = indexOfDeadline + 1; i < words.length; ++i) {
+                    by.append(" ").append(words[i]);
+                }
+                Task deadline = new Deadline(deadlineName.toString().trim(), by.toString().trim());
+                tasks.add(deadline);
+                reply = new StringBuilder(" added: " + deadline);
+                break;
+            case "event":
+                StringBuilder eventName = new StringBuilder();
+                StringBuilder startTime = new StringBuilder();
+                StringBuilder endTime = new StringBuilder();
+                int indexOfStartTime = Arrays.asList(words).indexOf("/from");
+                int indexOfEndTime = Arrays.asList(words).indexOf("/to");
+                for (int i = 1; i < indexOfStartTime; ++i) {
+                    eventName.append(" ").append(words[i]);
+                }
+                for (int i = indexOfStartTime + 1; i < indexOfEndTime; ++i) {
+                    startTime.append(" ").append(words[i]);
+                }
+                for (int i = indexOfEndTime + 1; i < words.length; ++i) {
+                    endTime.append(" ").append(words[i]);
+                }
+                Task event = new Event(eventName.toString().trim(), startTime.toString().trim(), endTime.toString().trim());
+                tasks.add(event);
+                reply = new StringBuilder(" added: " + event);
+                break;
+            }
+            System.out.println(HORIZONTAL_LINE + reply + HORIZONTAL_LINE);  // Output reply
+        }
+    }
+
+    /**
+     * Checks if Task Index input is in range of the current list of tasks
+     *
+     * @param tasks List of Tasks
+     * @param taskIndex Index of Task being validated
+     * @return true if Task is valid, false if Task is invalid
+     */
+    private static boolean validateTask(int tasks, int taskIndex) {
```

Comment:
Boolean methods can be phrased as a question, for example isATask.

Reply:
Thanks! I've renamed the boolean method to `isAValidTask`.